otterai/main.py

```diff
@@ -354,28 +350,25 @@ def main():
     pr_number = int(os.getenv('GITHUB_EVENT_NUMBER'))
     extra_prompt = os.getenv('INPUT_EXTRA_PROMPT', '')
     workspace = os.getenv('GITHUB_WORKSPACE', '.')
-
+    
     logging.info("🦦 Dr. OtterAI starting code review...")
+
+    # Handle GitHub operations
+    g = Github(github_token)
+    repo = g.get_repo(repo)
+    pr = repo.get_pull(pr_number)
+    get_commit = repo.get_commit(pr.head.sha)
     
     # Generate project context
     project_context = generate_review_context(workspace)
 
     # Get PR changes
-    diff_files = get_pr_diff(github_token, repo, pr_number)
+    diff_files = get_pr_diff(repo, pr)
```

Comment:
⚙️ Ensure that the `get_pr_diff` function handles potential exceptions when fetching the PR diff to avoid runtime errors.